x/ibc/03-connection/keeper/handshake.go

```diff
@@ -203,7 +199,8 @@ func (k Keeper) ConnOpenConfirm(
 		)
 	}
 
-	expectedConsensusState, found := k.clientKeeper.GetConsensusState(ctx, connection.ClientID, consensusHeight)
+	// TODO: No verification required?
```

Comment:
Sounds like we should verify here?

Reply:
I don't think so, this should never be written in response to any external messages, it's just data from Tendermint.